be/src/exec/pipeline/sink/iceberg_table_sink_operator.cpp

```diff
@@ -113,7 +112,14 @@ Status IcebergTableSinkOperator::push_chunk(RuntimeState* state, const ChunkPtr&
         const std::vector<std::string>& partition_column_names = _iceberg_table->partition_column_names();
         std::vector<std::string> partition_column_values;
         for (const ColumnPtr& column : partitions_columns) {
-            partition_column_values.emplace_back(_value_to_string(column));
+            std::string partition_value;
+            if (column->is_nullable()) {
+                const auto& nullable_column = std::static_pointer_cast<NullableColumn>(column);
```

Comment:
This is dangerous, const-null-column is_nullable(), then cast will crash

Reply:
> This is dangerous, const-null-column is_nullable(), then cast will crash

Is there any util function to use in this case?

Reply:
@zombee0 Is there any util function to use in this case?

Reply:
I have banned the null column.